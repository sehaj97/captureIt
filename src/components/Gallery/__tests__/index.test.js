import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Gallery from '..';

afterEach(cleanup);

describe('Gallery component', () => {
    //renders About test

    // First Test
  it('renders', () => {
    render(<Gallery />);
  });

  it('matches snapshot DOM node structure', () => {
    // render About
    const { asFragment } = render(<Gallery />);
    expect(asFragment()).toMatchSnapshot();
  });

  })