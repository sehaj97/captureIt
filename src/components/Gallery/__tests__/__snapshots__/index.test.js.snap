// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <section>
    <h1>
      NAME
    </h1>
    <p>
      DESCRIPTION
    </p>
  </section>
</DocumentFragment>
`;
